examples/plot_metrics_with_error.py

```diff
@@ -0,0 +1,163 @@
+# %%
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+"""
+================================
+Plotting Metrics with Error Bars
+================================
+"""
+# %%
+# Load and preprocess the data set
+# --------------------------------
+# We download the data set using `fetch_adult` function in `fairlearn.datasets`.
+# We start by importing the various modules we're going to use:
+#
+
+import numpy as np
+import pandas as pd
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.metrics import recall_score
+from sklearn.datasets import fetch_openml
+from sklearn.metrics import confusion_matrix
+from fairlearn.metrics import MetricFrame, ErrorPlotter
+
+
+data = fetch_openml(data_id=1590, as_frame=True)
+X = pd.get_dummies(data.data)
+y_true = (data.target == '>50K') * 1
+sex = data.data['sex']
+
+# %%
+# Error Metrics
+# -------------
+# We have many different choices for error metrics. In this notebook we'll just be using a
+# Normal approximation interval (symmetric) and a Wilson score interval (asymmetric).
+#
+# https://en.wikipedia.org/wiki/Binomial_proportion_confidence_interval#Wilson_score_interval
+# https://en.wikipedia.org/wiki/Binomial_proportion_confidence_interval#Normal_approximation_interval
+#
+#
+
+z_score = 1.959964
+digits = 4
+
+
+def recall_normal_err(y_t, y_p):
+    assert len(y_t) == len(y_p)
+    tn, fp, fn, tp = confusion_matrix(y_t, y_p).ravel()
+    error = compute_error_metric(tp/(tp+fn), tp + fn, z_score=z_score)
+    return error
+
+
+def wilson(p, n, digits=digits, z=z_score):
```

Comment:
Would these not be useful to define in `fairlearn.metrics`?

Reply:
Some of them (like this Wilson bound).... really they belong somewhere like `sklearn`, though

Reply:
Then shall we perhaps talk to `sklearn` folks about contributing it there? I don't mind having it here temporarily until `sklearn` has a release out with it, of course. @adrinjalali any thoughts?

Reply:
This sounds more like something that would be on either `scipy.stats` or `statsmodels` (which [already has a function called `proportion_confint`](https://www.statsmodels.org/stable/generated/statsmodels.stats.proportion.proportion_confint.html!))

Reply:
Great! Do we want to use the ones that are available from elsewhere then and only redefine the ones that don't exist yet?

Reply:
I second roman here.